examples/cookie/client.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/erikdubbelboer/fasthttp"
+)
+
+func main() {
+	req, res := fasthttp.AcquireRequest(), fasthttp.AcquireResponse()
+	req.SetRequestURI("http://localhost:8080")
+
+	err := fasthttp.Do(req, res)
+	if err != nil {
+		panic(err)
+	}
+
+	// We can get the cookie from diferents ways.
+	// 1. Using 'key' parameter. Fasthttp will search using key in cookie field.
+	cookie := fasthttp.AcquireCookie()
+	cookie.SetKey("key")
+
+	res.Header.Cookie(cookie)
+	fmt.Printf("You cookie is '%s' found using '%s' key\n", cookie.Cookie(), cookie.Key())
+
+	cookie.Reset()
+
+	// 2. You can parse request header 'Set-Cookie' field.
+	value := res.Header.Peek("Set-Cookie")
```

Comment:
I don't think this will work correctly if there are multiple `Set-Cookie` headers?

Reply:
You are right. I just want to show it with one key. One problem of fasthttp is that we have to iterate cookies with VisitAllCookie and add it into an array of Cookies (CookieJar in Golang std library). So... I can create CookieJar and modify this example by using it :D